build_runner/lib/src/build/library_cycle_graph/library_cycle_graph_loader.dart

```diff
@@ -341,24 +341,21 @@ class LibraryCycleGraphLoader {
   ///
   /// A [_graphs] entry will be created for each ID in [newCycles].
   void _buildGraphs(int phase, {required List<LibraryCycle> newCycles}) {
-    // Build lookup from ID to [LibraryCycle] including new and existing cycles.
```

Comment:
Am I reading this right, that this boils down to not pre-populating the map?

Reply:
Yes, exactly :)